### ISSUE TITLE
Exporting hyperspectral data from h5browser

### ISSUE THREAD

**Issue by LMSC-SMarinoni** [PyMoDAQ/PyMoDAQ#90]
Hello!

At EPFL-LMSC we're acquiring hyperspectral maps with PyMoDAQ, and we run into a problem during the export from .h5 to .txt using the h5browser. It looks like only 1D and 2D data can be exported, as the h5browser calls the numpy.savetxt() function.

For the moment, we solved the issue by applying numpy.ndarray.flatten() on the data (saving the data shape in the file header to reconstruct the hyperspectral map later on), but there might be a better solution that could be implemented for the entire community.

Here are the logbook lines:
```
2022-12-21 12:19:07,634 - pymodaq.h5modules - ERROR - Expected 1D or 2D array, got 3D array instead
Traceback (most recent call last):
  File "C:\Users\m\Miniconda3\envs\pmodaq\lib\site-packages\pymodaq\daq_utils\h5modules.py", line 2286, in export_data
    self.h5utils.export_data(self.current_node_path, str(file))
  File "C:\Users\m\Miniconda3\envs\pmodaq\lib\site-packages\pymodaq\daq_utils\h5modules.py", line 1976, in export_data
    np.savetxt(filesavename,
  File "<__array_function__ internals>", line 180, in savetxt
  File "C:\Users\m\Miniconda3\envs\pmodaq\lib\site-packages\numpy\lib\npyio.py", line 1512, in savetxt
    raise ValueError(
ValueError: Expected 1D or 2D array, got 3D array instead
```

Cheers! :)

**Comment by seb5g** [comment]
Hi,
Well it doesn't make any sense to export heavy load data as txt while you have them as packed binary data in the h5 file. I would suggest to directly use the data as is from software able to deal with hdf5: python or others. Export to txt for 1D or 2D is just a helper for people who are used to this kind of data management, but it is really a bad practice...

What do you want to do with the txt files?

**Comment by LMSC-SMarinoni** [comment]
Hello,

Thanks for the reply!
Some of us are just used to this kind of data format and work with scripts based on it.
We will surely try to change our habits towards better practices. But then, I would suggest to disable/hide the possibility of heavy (1D, 2D) data export in .txt format. Otherwise, other users will try this possibility and encounter the same problem.

**Comment by LMSC-NTappy** [comment]
Solved with #93
